README.md

```diff
@@ -61,4 +62,15 @@ sudo psql -U root postgres < database/schema.sql && node database/dataSeed.js
 ## Run webpack
 (first switch to superuser) sudo su -
 (then go back a directory and go to home/ec2-user/fetsyShopping)
-npm run react-dev
\ No newline at end of file
+npm run react-dev
+
+<a name="crud"></a>
+# CRUD Documentation
+
+| Methods  | URL Endpoints | Actions |
+| :------- | :------------ | :------ |
+| GET | /shopping/items  | Retrieve all items available for sale |
+| GET | /shopping/items/:itemId  | Retrieve a single item |
+| POST | /shopping/items | Add a new item for sale |
```

Comment:
Maybe include the shape of the data needed to properly hit this endpoint or other endpoints. took me a sec to figure out how your post and put are receiving the correct data. Good work though, I like the organization!

Reply:
Great idea - adding those details now.